images/macos/software-report/SoftwareReport.Common.psm1

```diff
@@ -1,12 +1,12 @@
 function Get-BashVersion {
     $version = bash -c 'echo ${BASH_VERSION}'
-    return "Bash $version"
+    return "$version"
```

Comment:
nit: `return $version`. The same comment in some other places.

Reply:
yes, corrected